.gitignore

```diff
@@ -1,9 +1,10 @@
 .bundle/
+Gemfile.lock
 log/*.log
 pkg/
-test/dummy/db/*.sqlite3
-test/dummy/db/*.sqlite3-journal
-test/dummy/db/*.sqlite3-*
-test/dummy/log/*.log
-test/dummy/storage/
-test/dummy/tmp/
+spec/dummy/db/*.sqlite3
+spec/dummy/db/*.sqlite3-journal
+spec/dummy/db/*.sqlite3-*
+spec/dummy/log/*.log
+spec/dummy/storage/
+spec/dummy/tmp/
```

Comment:
aren't you using a test directory not a spec directory? not sure I understand if this is gitignoring the right stuff